sdk/release_notes.md

```diff
@@ -6,7 +6,7 @@
 
 ### Microsoft.Azure.Functions.Worker.Sdk 1.15.1 (meta package)
```

Comment:
Do we also replace version number with `<version>`?

Reply:
Good catch. Fixed that in the latest iteration.